website/docs/r/workspace.html.markdown

```diff
@@ -67,6 +67,8 @@ The following arguments are supported:
   in a VCS push. If enabled, the working directory and trigger prefixes describe a set of 
   paths which must contain changes for a VCS push to trigger a run. If disabled, any push will 
   trigger a run. Defaults to `true`.
+* `global_remote_state` - (Optional) Whether to allow remote state access to this workspace. It defaults to `true`.
```

Comment:
The other arguments here don't follow the convention (🙄) but typically you see the same description as the associated resource argument, but tweaked slightly to indicate that it's read-only:

```suggestion
* `global_remote_state` - (Optional) Whether the workspace allows all workspaces in the organization to access its state data during runs. If false, then only specifically approved workspaces can access its state (`remote_state_consumer_ids`).
```